pcl2_to_costmap:
  ros__parameters:
    semantic_map_frame: "semantic_map"
    map_frame: "map"